api/envoy/service/auth/v2alpha/external_auth.proto

```diff
@@ -2,12 +2,15 @@ syntax = "proto3";
 
 package envoy.service.auth.v2alpha;
 
+import "envoy/service/auth/v2/external_auth.proto";
```

Comment:
This a strange dependency relationship, having the v2alpha depend on the v2..

Reply:
Yeah, that strange relationship is the root reason for all the special cases.